src/ui/public/vislib/visualizations/_map.js

```diff
@@ -272,10 +272,22 @@ define(function (require) {
     TileMapMap.prototype._createMap = function (mapOptions) {
       if (this.map) this.destroy();
 
+
+      if (this._attr.wms.enabled) {
+        _.assign(mapOptions, {
+          minZoom: 1,
+          maxZoom: 18
```

Comment:
Maybe these should respect the server config if the max is greater than 18? In other words:


`maxZoom: Math.max(18, mapOptions.maxZoom)`